src/content/en/tools/chrome-devtools/javascript/get-started.md

```diff
@@ -0,0 +1,274 @@
+
+1. Open DevTools on the demo by pressing
+   <kbd>Command</kbd>+<kbd>Option</kbd>+<kbd>I</kbd> (Mac) or
+   <kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Windows, Linux).
+1. Click the **Sources** tab.
+1. Click **Event Listener Breakpoints** to expand the section. DevTools reveals
+   a list of expandable event categories, such as **Animation** and
+   **Clipboard**.
+1. Next to the **Mouse** event category, click **Expand** ![Expand
+   icon](/web/tools/chrome-devtools/images/expand.png){: .devtools-inline}.
+   DevTools reveals a list of mouse events, such as **click**,
+   with checkboxes next to them.
+1. Check the **click** checkbox.
+1. Back on the demo, click **Add Number 1 and Number 2** again. DevTools
+   pauses the demo and highlights a line of code in the **Sources** panel.
+1. Check that DevTools is highlighting this line of code:
+
+       `function onClick() {`
+
+1. If it's highlighting any other line, keep pressing **Resume script
+   execution** ![Resume script execution][resume]{: .devtools-inline }
+   until DevTools highlights the line of code above. The highlighted line
+   represents the line of code that you're currently paused on.
+
+When you checked the **click** checkbox, you set up an event-based breakpoint on
+all `click` events. When *any* node is clicked, and that node has a `click`
+handler, DevTools automatically pauses on the first line of that node's
+`click` handler.
+
+Note: This is just one of the many types of breakpoints that DevTools offers.
+What breakpoint you should use depends on what type of issue you're debugging.
+
+[resume]: /web/tools/chrome-devtools/images/resume-script-execution.png
+
+## Step 3: Step through the code
+
+One common cause of bugs is when a script executes in the
+wrong order. Stepping through your code enables you to walk through your
+code's execution, one line at a time, and figure out exactly where it's
+executing in a different order than you expected. Try it now:
+
+1. On the **Sources** panel of DevTools, click **Step into next function
+   call** ![Step into next function call][into]{:.devtools-inline} to step
+   through the execution of the `onClick()` function, one line at a time.
+   DevTools highlights the following line of code:
+
+       `if (inputsAreEmpty()) {` 
+
+1. Click **Step over next function call** ![Step over next function
+   call][over]{:.devtools-inline}. DevTools executes `inputsAreEmpty()`
+   without stepping into it. Notice how DevTools skips a few lines of code.
+   This is because `inputsAreEmpty()` evaluated to false, so the `if`
+   statement's block of code didn't execute.
+
+That's the basic idea of stepping through code. If you look at the code in
+`get-started-1.js`, you can see that the bug is probably somewhere in the
+`updateLabel()` function. Rather than stepping through every line of code,
+you can use another type of breakpoint to pause the code closer to the
+probably location of the bug.
```

Comment:
Remove 'probably'. You could change it to 'probable', but I think removing it all together is a nicer alternative.